apps/src/applab/applab.js

```diff
@@ -1349,90 +1337,6 @@ function onInterfaceModeChange(mode) {
   requestAnimationFrame(() => showHideWorkspaceCallouts());
 }
 
-function onDataPreview(tableName) {
-  onColumnsChange(getSharedDatabase(), tableName, columnNames => {
-    getStore().dispatch(updateTableColumns(tableName, columnNames));
-  });
-  getPathRef(getSharedDatabase(), `storage/tables/${tableName}/records`).once(
-    'value',
-    snapshot => {
-      getStore().dispatch(updateTableRecords(tableName, snapshot.val()));
-    }
-  );
-}
-
-/**
- * Handle a view change within data mode.
- * @param {DataView} view
- */
-function onDataViewChange(view, oldTableName, newTableName) {
```

Comment:
onDataViewChange has been renamed to loadDataForView and refactored into loadDataView.js. This name better reflects the usage in Datablock Storage in particular, but also gives a better sense of what the firebase code does.

Most of the code seen below is pulled out into Firebase-specific methods in firebaseStorage.js